apps/tangle-dapp/components/UnbondingStatsItem/UnbondingStatsItem.tsx

```diff
@@ -0,0 +1,55 @@
+'use client';
+
+import { notificationApi } from '@webb-tools/webb-ui-components';
+import { type FC, Fragment, useMemo } from 'react';
+
+import useUnbondingRemainingErasSubscription from '../../data/NominatorStats/useUnbondingRemainingErasSubscription';
+import { NominatorStatsItem } from '../NominatorStatsItem';
+
+const UnbondingStatsItem: FC<{ address: string }> = ({ address }) => {
+  const {
+    data: unbondingRemainingErasData,
+    error: unbondingRemainingErasError,
+  } = useUnbondingRemainingErasSubscription(address);
+
+  const unbondingRemainingErasTooltip = useMemo(() => {
+    if (unbondingRemainingErasError) {
+      notificationApi({
+        variant: 'error',
+        message: unbondingRemainingErasError.message,
+      });
+    }
+
+    if (!unbondingRemainingErasData?.value1) return null;
+
+    if (unbondingRemainingErasData.value1.length === 0) {
+      return 'You have no unbonding tokens.';
+    }
+
+    const elements = unbondingRemainingErasData.value1.map((era, index) => {
+      return (
+        <Fragment key={index}>
+          <div className="text-center mb-2">
+            <p>
+              {era.remainingEras > 0 ? 'Unbonding' : 'Unbonded'} {era.amount}
+            </p>
+            {era.remainingEras > 0 && <p>{era.remainingEras} eras remaining</p>}
+          </div>
+        </Fragment>
+      );
+    });
+
+    return <>{elements}</>;
+  }, [unbondingRemainingErasError, unbondingRemainingErasData?.value1]);
+
+  return (
+    <NominatorStatsItem
+      title="Unbonding tTNT"
```

Comment:
We should have a constant or function to get the token unit from a centralized place, so that we can easily update this once the mainnet launches. I believe there is a constant in `constants/polkadot`